### DISCUSSION TITLE
Change background to bg for feature parity with styled-system

### DISCUSSION THREAD

**Discussion by Slooowpoke** [spicy-ui/styled-system#5] [Q&A] [answered]
This package looks pretty good and it's nice to see it being updated but I think if it was ever to work as a viable option it needs to stick to the original styled-system package. What was the reason for changing it to background?

**Comment by South-Paw** [comment, marked as answer]
Hi there,

`bg` in styled system maps to the `background-color` CSS property if I remember correctly.

As the `background` CSS property has [other properties](https://developer.mozilla.org/en-US/docs/Web/CSS/background), I thought it'd be best to map them more intuitively to match CSS.

`bgColor` also exists for the `background-color` property, but `bg` will still work as expected (just be `background: #fff` instead of `background-color: #fff`)

**Reply by Slooowpoke** [reply]
Right of course! I don’t know why I assumed it would break compatibility completely. Thank you.
